CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### [0.24.10](https://github.com/plus3it/tardigrade-ci/releases/tag/0.24.9)
+
+**Released**: 2023.08.17
+
+**Summary**:
+
+*   Add new makefile target 'pullrequest' to run formatters, lint and documentation in one call
+
+
```

Comment:
Can you add the `* Updates tool versions:` section?  Review the dependabot commits since the last release to get the versions.